### ISSUE TITLE
gue1-spclient.spotify.com breaks Spotify on Google Homes

### ISSUE THREAD

**Issue by p1r473** [ShadowWhisperer/BlockLists#73]
gue1-spclient.spotify.com breaks Spotify on Google Homes

The music just wont load

Thanks

**Comment by ShadowWhisperer** [comment]
Sorry for the long delay. I had to test this on my parents speaker, as I do not have one.

It is the original 1st generation Google home speaker. [This One](http://mobileimages.lowes.com/productimages/4c017128-ea41-421d-9fcc-8186e087f9b4/15673340.jpgl)
I have it linked to a Google Pixel 4a phone, and my free Spotify account. The speaker, the phone, and the Spotify app are fully up to date.

I see the domain call out, and return blocked.  Music plays without issue. I tried a factory reset on the speaker and reinstalling the Google Home app. It's been playing in the kitchen for the last few hours.

All wireless traffic is routed through my Debian server. DNS is redirected to it. (Setting a custom DNS on a device will still go to the server) Known DoH domains/IPs are blocked.

------------------------

What generation of Google Home are you using?
